common/spring-cloud-stream-modules-test-support/src/main/java/org/springframework/cloud/stream/modules/test/file/remote/TestFtpServer.java

```diff
@@ -159,14 +161,17 @@ public void before() throws Throwable {
 
 
 	@PreDestroy
-	public void after() {
-		this.server.stop();
+	public void after() throws Exception {
+		stopServer();
 		this.ftpTemporaryFolder.delete();
 		if (localTemporaryFolder != null) {
 			this.localTemporaryFolder.delete();
```

Comment:
This is done for you by `TemporaryFolder` already.
Seems it is missused though, I don't see no `@Rule`